_summaries/kingma2014adam_2.md

```diff
@@ -1,19 +1,28 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: kingma2014adam
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: ishank-arora
+score: 9
 ---
 
-TODO: Summarize the paper:
 * What is the core idea?
+  * Introduces the Adam (Adaptive Moment Estimation) algorithm.
+    * First-order gradient-based optimization of stochastic objective functions
+  * Algorithm based on adaptive estimates
+  * Computationally efficient
+  * Intended to combine the advantages of AdaGrad and RMSProp. 
+  * Moment estimates bias toward 0 without bias correction
 * How is it realized (technically)?
+  * <img src="kingma2014adam_2_1.png" alt="kingma2014adam_2_1" style="zoom: 50%;" />
 * How well does the paper perform?
+  * Very well
+  * ![kingma2014adam_2_2](kingma2014adam_2_2.png)
 * What interesting variants are explored?
+  * AdaMax - based on the infinity norm
```

Comment:
It might not be very important, but I think Nadam and AMSGrad also deserve places on this list. Even by now, there are several papers that try to propose a better adam-like function, thus I personally find it interesting to compare the differences between each of them.